docfx.json

```
@@ -15,11 +15,11 @@
         "src": "docs",
         "dest": ".",
         "exclude": [
-          "samples/**/*.*",
-          "**/snippets/**",
-          "**/includes/**",
-          "**/util/**",
-          "***/contributing.md"
+            "samples/**/*.*",
+            "**/snippets/**",
+            "**/includes/**",
+            "**/util/**",
+            "***/contributing.md"
```

nit: Revert indentation to be consistent with the rest of the file.

```suggestion
          "samples/**/*.*",
          "**/snippets/**",
          "**/includes/**",
          "**/util/**",
          "***/contributing.md"
```